ext/pdo_mysql/tests/bug_38546.phpt

```diff
@@ -230,14 +230,14 @@ Array
     [3] => 2
 )
 
-Warning: PDOStatement::execute(): SQLSTATE[HY000]: General error: 1366 Incorrect integer value: 'true' for column 'some_bool_1' at row 1 in %s
+Warning: PDOStatement::execute(): SQLSTATE[%s]: %s: 1366 Incorrect integer value: 'true' for column %s at row 1 in %s
```

Comment:
I would like to see here a regex pattern with original `HY000`

Reply:
I really didn't know how to handle this, and I asked for help. This was recommended by CMB. If you have another option I can change it.

Reply:
What you did is already right, ignore the comment.

Reply:
oh, ok, it is for bug test only...